contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -3442,6 +3442,10 @@ RETURNS sys.NVARCHAR(128)  AS 'babelfishpg_tsql' LANGUAGE C STABLE;
 CREATE OR REPLACE FUNCTION sys.host_name()
 RETURNS sys.NVARCHAR(128)  AS 'babelfishpg_tsql' LANGUAGE C IMMUTABLE PARALLEL SAFE;
 
+CREATE OR REPLACE FUNCTION sys.host_id()
+RETURNS sys.VARCHAR(10)  AS 'babelfishpg_tsql' LANGUAGE C IMMUTABLE PARALLEL SAFE;
```

Comment:
https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/t-sql/functions/host-id-transact-sql.md says its return type is `char(10)`. Is there a reason to use `varchar(10)`?

Reply:
When using sys.CHAR(10), the following error is raised when creating the extension:

```
jdbc_testdb=# CREATE EXTENSION IF NOT EXISTS "babelfishpg_tds" CASCADE;
NOTICE:  installing required extension "babelfishpg_tsql"
NOTICE:  installing required extension "uuid-ossp"
NOTICE:  installing required extension "babelfishpg_common"
ERROR:  type modifier cannot be specified for shell type "sys.char"
```